environment.yml

```diff
@@ -29,7 +29,7 @@ dependencies:
   - fire
   - fsspec
   - gdal >=3.9.0
-  - importlib_metadata <8  # https://github.com/ESMValGroup/ESMValTool/issues/3699 only for Python 3.10/11 and esmpy<8.6
+  - importlib_metadata <8  # https://github.com/ESMValGroup/ESMValTool/issues/3699 only for Python 3.11 and esmpy<8.6
```

Comment:
sorry Many, having spoken about NCL just now and @bouweandela having fixed it for now, this can be removed as well